Assets/Scripts/API/ApiMethods.MultiMirror.cs

```diff
@@ -97,15 +97,6 @@ public static void WallpaperSymmetrySkewX(float x, float y)
             PointerManager.m_Instance.CalculateMirrors();
         }
 
-        [ApiEndpoint("symmetry.colorshift.enable",
```

Comment:
Is there any possibility someone could be using this? Do you need some kind of deprecation handling?

Reply:
Highly unlikely considering the number of known API users. And I don't want too commit to API stability for stuff that's only made it as far as beta.

Reply:
sure (it was more a technical question than a policy one :-) )

Reply:
merge away!